src/components/App.js

```diff
@@ -194,11 +198,13 @@ class App extends Component<Props, State> {
   }
 
   setOrientation() {
-    const orientation = verticalLayoutBreakpoint.matches
-      ? "horizontal"
-      : "vertical";
-    if (isVisible()) {
-      this.props.setOrientation(orientation);
+    // If the orientation does not match (if it is not visible) it will
+    // not setOrientation, or if it is the same as before, calling
+    // setOrientation will not cause a rerender.
+    if (horizontalLayoutBreakpoint.matches) {
```

Comment:
Same as above - does it seem feasible to just address the horizontal layout and treat everything else as vertical?

Reply:
We could, but, that would cause a re-render every time we change away from the debugger tab, and another re-render when we change back.